packages/pyright-internal/src/common/shared.ts

```diff
@@ -0,0 +1,19 @@
+/*
+ * shared.ts
```

Comment:
Hmm, this isn't a very good name for a module. It's already located in a "common" directory, which implies that it's shared. The file name should describe what it does. Consider choosing a better name.

Reply:
any suggestion? how about `sharedSetup.ts` ?

Reply:
How about `asyncInitialization.ts`?